pegasus/sites.v3/code.org/views/hourofcode_sidebar.haml

```diff
@@ -0,0 +1,35 @@
+:ruby
+  corporate_donors = [{name_s: 'Amazon', url_s: 'http://www.amazon.com'}, {name_s: 'Microsoft', url_s: 'http://www.microsoft.com'}, {name_s: 'Google', url_s: 'http://www.google.com'}, {name_s: 'Juniper Networks Foundation Fund', url_s: 'http://www.juniper.net'}, {name_s: 'Salesforce', url_s: 'http://www.salesforce.com'}, {name_s: 'LinkedIn', url_s: 'http://www.linkedin.com'}, {name_s: 'JP Morgan Chase & Co', url_s: 'http://www.jpmorganchase.com/'}, {name_s: 'Allen & Co', url_s: ''}, {name_s: 'Best Buy', url_s: 'http://www.bestbuy.com/'}]
```

Comment:
Oh good idea. Where should I put the csv file? In the same directory or
somewhere like lib or bin?

On Tuesday, July 19, 2016, Brendan Reville <notifications@github.com> wrote:

> In pegasus/sites.v3/code.org/views/hourofcode_sidebar.haml
> <https://github.com/code-dot-org/code-dot-org/pull/9609#discussion_r71467654>
> :
>
> > @@ -0,0 +1,35 @@
> > +:ruby
> > +  corporate_donors = [{name_s: 'Amazon', url_s: 'http://www.amazon.com'}, {name_s: 'Microsoft', url_s: 'http://www.microsoft.com'}, {name_s: 'Google', url_s: 'http://www.google.com'}, {name_s: 'Juniper Networks Foundation Fund', url_s: 'http://www.juniper.net'}, {name_s: 'Salesforce', url_s: 'http://www.salesforce.com'}, {name_s: 'LinkedIn', url_s: 'http://www.linkedin.com'}, {name_s: 'JP Morgan Chase & Co', url_s: 'http://www.jpmorganchase.com/'}, {name_s: 'Allen & Co', url_s: ''}, {name_s: 'Best Buy', url_s: 'http://www.bestbuy.com/'}]
>
> Can we maybe move these to a csv or yml file and then parse it with Ruby?
> It's pretty simple code and will be easier to review.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/9609/files/e44108627068de1338ec0419d5de3f7287cb90b3#r71467654>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB1OEgP1XCnTdGMUUFyTMkzSbEprMnVbks5qXbNEgaJpZM4JQVNc>
> .
>


-- 
Please excuse typos, sent from mobile